python/aswfdocker/utils.py

```diff
@@ -0,0 +1,67 @@
+import os
+import subprocess
+import datetime
+from aswfdocker import constants
+
+
+def get_current_branch() -> str:
+    return subprocess.check_output("git rev-parse --abbrev-ref HEAD", encoding="UTF-8")
+
+
+def get_current_sha() -> str:
+    return subprocess.check_output("git rev-parse --short HEAD", encoding="UTF-8")
+
+
+def get_current_date() -> str:
+    # 2020-01-19T08:04:51Z
+    return datetime.datetime.now().isoformat(timespec="seconds") + "Z"
+
+
+def get_docker_org(repo_uri: str, source_branch: str) -> str:
+    if not source_branch and not repo_uri:
+        return constants.TESTING_DOCKER_ORG
+    if (
+        source_branch == "refs/heads/master"
+        and repo_uri == "https://github.com/AcademySoftwareFoundation/aswf-docker"
+    ):
+        docker_org = constants.PUBLISH_DOCKER_ORG
+    elif source_branch in ("refs/heads/testing", ""):
+        docker_org = constants.TESTING_DOCKER_ORG
+    else:
+        docker_org = constants.FAKE_DOCKER_ORG
+    return docker_org
+
+
+def get_docker_push(repo_uri: str, source_branch: str) -> str:
+    if (
+        source_branch == "refs/heads/master"
+        and repo_uri == "https://github.com/AcademySoftwareFoundation/aswf-docker"
```

Comment:
Similar comment as previously, use a configuration file for the ASWF github path